### PR TITLE
ci: use `tui` prefix for exported functions

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [x] Build or CI related changes
- [ ] Documentation content changes

Part of #1764

### CODE REVIEW

**Review by waterplea — COMMENTED** [review]
Can we check if all these input assertions can now just be inline arrow functions? It is exported just so AOT compilation work, but maybe it's not necessary anymore. Or maybe it won't be once we publish in Ivy 🤔 In this case we can skip them until 3.0, they are internal functions exported just so compiler works, nobody uses them externally.

File: projects/addon-charts/components/bar-chart/bar-chart.component.ts [hunk 1]
```diff
@@ -12,14 +12,19 @@ import {
...
 
+/**
+ * @deprecated: use {@link tuiValueAssertion} instead
+ */
+// eslint-disable-next-line @typescript-eslint/naming-convention
 export function valueAssertion(value: ReadonlyArray<readonly number[]>): boolean {
-    const valid = value.every(array => array.length === value[0].length);
-
-    return valid;
+    return value.every(array => array.length === value[0].length);
```
[the remark this answers is not in the document]
Comment (splincode) [reply projects/addon-charts/components/bar-chart/bar-chart.component.ts:20]: Do you mean, we can delete `@tuiDefaultProp(valueAssertion, VALUE_ERROR)` line?

**Review by waterplea — APPROVED** [review]

File: projects/addon-charts/utils/control-point.ts [hunk 2]
```diff
@@ -19,3 +23,5 @@ export function controlPoint(
 
     return [x, y];
 }
+
+export const tuiControlPoint = controlPoint;
```
[the remark this answers is not in the document]
Comment (splincode) [reply projects/addon-charts/utils/control-point.ts:27]: in the next PR

**Review by nsbarsukov — CHANGES_REQUESTED** [review]

File: projects/kit/components/input-phone-international/const/countries.ts [hunk 3]
```diff
@@ -254,6 +255,8 @@ export function countriesMasksReducer(
     return acc;
 }
 
+export const tuiCountriesMasksReducer = countriesMasksReducer;
```
[the remark this answers is not in the document]
Comment (splincode) [reply projects/kit/components/input-phone-international/const/countries.ts]: fixed

File: projects/core/utils/format/pluralize.ts [hunk 4]
```diff
@@ -24,3 +26,5 @@ export function pluralize(value: number, [one, few, many]: TuiPluralize): string
 
     return many;
 }
+
+export const tuiPluralize = pluralize;
```
[the remark this answers is not in the document]
Comment (splincode) [reply projects/core/utils/format/pluralize.ts]: fixed

File: projects/addon-charts/utils/draw-line.ts [hunk 5]
```diff
@@ -1,3 +1,7 @@
+/**
+ * @deprecated: use {@link tuiControlPoint} instead
```
[the remark this answers is not in the document]
Comment (splincode) [reply projects/addon-charts/utils/draw-line.ts]: fixed

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**